python/python/lancedb/remote/table.py

```diff
@@ -564,6 +565,21 @@ def compact_files(self, *_):
             "compact_files() is not supported on the LanceDB cloud"
         )
 
+    def optimize_indices(self, **kwargs):
+        """optimize_indices() is not supported on the LanceDB cloud"""
+        raise NotImplementedError(
+            "optimize_indices() is not supported on the LanceDB cloud"
+        )
+
+    def optimize(
+        self,
+        *,
+        cleanup_older_than: Optional[timedelta] = None,
+        delete_unverified: bool = False,
+    ):
+        """optimize() is not supported on the LanceDB cloud"""
+        raise NotImplementedError("optimize() is not supported on the LanceDB cloud")
+
```

Comment:
Can we make a slight change from `"optimize() is not supported on the LanceDB cloud"` to `"optimize() is not supported on the LanceDB cloud.  Indices are optimized automatically."`

Reply:
fixed